capabilities.json

```diff
@@ -132,15 +132,28 @@
         "label": {
             "displayName": "Center Label",
             "properties": {
-                "fill": {
-                    "displayName": "Fill",
+                "show": {
+                    "type": {
+                        "bool": true
+                    }
+                },
+                "color": {
```

Comment:
@ignatvilesov  "Detailed labels" - is "labelS" object. "Center Label" is "label" - didn't use before.  So, I will change name object(or create new) to exclude confusing. And we can keep "label" object in capabilities.json to exclude breaking of visual.